examples/azure/managed-disks/build.sh

```diff
@@ -11,8 +11,16 @@ import klb/azure/disk
 import klb/azure/vnet
 import klb/azure/nsg
 import klb/azure/route
+import klb/azure/snapshot
 import config.sh
 
+fn addsuffix(name) {
+	# Providing true uniqueness with the limits on the names is pretty hard :-)
+	s <= head -n1 /dev/urandom | md5sum | tr -dc A-Za-z0-9 | cut -b 1-10
```

Comment:
It is just an example, I did the fastest thing that I could :-P. On the final backup the timestamp should be better